File: VB.NET/0001multiples_of_3_or_5.vb
REM 
REM .NET VERSION 7.0.410
REM 0001multiples_of_3_or_5.vb
REM 
REM Copyright 2025 @GitHub JoergEm
REM 
REM This program is WITHOUT ANY WARRANTY; without even the implied
REM warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
REM 

Module Module1
    Sub Main()
        Dim summe As Integer = 0
        For nummer As Integer = 1 To 999
            If nummer Mod 3 = 0 Or nummer Mod 5 = 0 Then
                summe += nummer
            End If
        Next
        Console.WriteLine(summe)
    End Sub
End Module

File: Octave/0001multiples_of_3_or_5.m
%{
% octave version 8.4.0
% 0001multiples_of_3_or_5.m
% Copyright 2025 @GitHub JoergEm
%
% This program is WITHOUT ANY WARRANTY; without even the implied
% warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
%}


summe = 0;
for nummer = 1:999
    if mod(nummer, 3) == 0 || mod(nummer, 5) == 0
        summe = summe + nummer;
    end
end

fprintf('%d\n', summe);
